packages/apps/tools/src/components/Global/AddNetworkModal/AddNetworkModal.tsx

```diff
@@ -15,9 +15,9 @@ import { useForm } from 'react-hook-form';
 import * as z from 'zod';
 
 const schema = z.object({
-  label: z.string().trim(),
-  networkId: z.string().trim(),
-  api: z.string().trim(),
+  label: z.string().trim().min(1),
+  networkId: z.string().trim().min(1),
+  api: z.string().trim().min(1),
```

Comment:
You could perhaps reuse [this](https://github.com/kadena-community/kadena.js/blob/main/packages/apps/tools/src/pages/transactions/cross-chain-transfer-finisher/index.tsx#L65) for validating this field?

Reply:
it will fail for custom networks (localhost:8080 or something)

Reply:
Ah you're right, that wouldn't work, my bad